.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,24 +1,24 @@
 ---
 name: Bug report
 about: Create a report to help us improve
-
+label: bug
```

Comment:
Would it make sense to add the "not ready for work" label here so we can triage?